### ISSUE TITLE
Rebuild, rewrite or test tests for travis builds

### ISSUE THREAD

**Issue by timtrice** [ropensci/rrricanes#25]
Referencing Issue #23, tests were failing in travis for some reason. Exact error thrown:

```
* checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  11: xml2::read_html(link)
  12: read_html.default(link)
  13: suppressWarnings(read_xml(x, encoding = encoding, ..., as_html = TRUE, options = options))
  14: withCallingHandlers(expr, warning = function(w) invokeRestart("muffleWarning"))
  15: read_xml(x, encoding = encoding, ..., as_html = TRUE, options = options)
  
  Error in UseMethod("read_xml") : 
    no applicable method for 'read_xml' applied to an object of class "factor"
  Calls: test_check ... %>% -> eval -> eval -> <Anonymous> -> read_html.default
  testthat results ================================================================
  OK: 56 SKIPPED: 0 FAILED: 1
  1. Error: Get products from storm archive (@test_filters.R#7) 
  
  Error: testthat unit tests failed
```

These tests passed locally. Session Info:

```
R version 3.4.0 (2017-04-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.2 LTS

Matrix products: default
BLAS: /usr/lib/libblas/libblas.so.3.6.0
LAPACK: /usr/lib/lapack/liblapack.so.3.6.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] Hurricanes_0.0.0.9002 testthat_1.0.2        magrittr_1.5          dplyr_0.5.0          

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.7       xml2_1.1.1        knitr_1.15.1      devtools_1.12.0   rvest_0.3.2       R6_2.2.0          stringr_1.1.0    
 [8] httr_1.2.1        tools_3.4.0       data.table_1.10.4 DBI_0.5-1         selectr_0.3-0     withr_1.0.2       htmltools_0.3.6  
[15] lazyeval_0.2.0    yaml_2.1.14       assertthat_0.1    digest_0.6.10     rprojroot_1.2     tibble_1.2        crayon_1.3.2     
[22] curl_2.6          memoise_1.1.0     evaluate_0.10     rmarkdown_1.5     stringi_1.1.2     compiler_3.4.0    backports_1.0.5  
[29] XML_3.98-1.6      lubridate_1.6.0  
```

Start from top, work down to reintroduce tests. Tests can be found in commit e7834ca.

**Comment by timtrice** [comment]
Tests reintroduced to `develop-0.1.0` in commit eb63d8b. That [build fails](https://travis-ci.org/timtrice/Hurricanes/builds/228863854). Created child branch `tests-0.1.0`.

**Comment by timtrice** [comment]
Ran `devtools::update_packages()` which matched the Travis environment. Was able to isolate the issue to  the datetime values extract from the text products being converted on insertion into dataframes. Changed default value of Dates in all dataframe skeletons from `character()` to `as.POSIXct(character(), tz = "UTC")`. This passes tests locally and in Travis.

Will create another issue for the timezones. Some text products do not use UTC and therefore there could be some problems with this. It may not show up in errors but rather just simply be stored as bad data. Will need to build tests for this.

**Comment by timtrice** [comment]
Closed in PR #33
